### PR TITLE
Added Microsoft Logo CSS Hover Project

### PR DESCRIPTION
**Pull request by parinzee**
Hey there! This PR addresses #14. I hope you find this satisfactory and merge this PR!

Regards,
Parin

### CODE REVIEW

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Great work so far @parinzee !
Just a few changes.

- Project folder name should be your name followed by project name. For example, `ParinzeeMicrosoftLogo`
- Put the gif file inside `/public/assets`

Everything else is fine. Do the mentioned changes and I'll be happy to merge.

**Comment by parinzee** [comment]
Thank you @SandeepKrSuman! I've made the requested changes. 😁

File: public/Projects/ParinzeeMicrosoftLogo/info.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     name: "Parinthapat Pengpun",
     ghurl: "https://github.com/parinzee",
-    imgName: "demo.gif",
+    imgName: "ParinzeeMicrosoftLogo.gif",
     folderName: "MicrsoftLogo",
```
Comment (SandeepKrSuman) [inline public/Projects/ParinzeeMicrosoftLogo/info.txt:5]:
```
folderName: "ParinzeeMicrosoftLogo",
```

**Review by SandeepKrSuman — APPROVED** [review]
Looks good! 👍

**Comment by parinzee** [comment]
Thank you very much! Sorry for the oversight there.

**Pull request merged**